### ISSUE TITLE
Cannot get one-day chart

### ISSUE THREAD

**Issue by a-maggioni** [WojciechZankowski/iextrading4j#19]
## Summary
I tried to run this example with a range of 1 day but I got an exception: [https://github.com/WojciechZankowski/iextrading4j/wiki/Chart](url)

## Details
My code is:
`final IEXTradingClient iexTradingClient = IEXTradingClient.create();`
`final List<Chart> chartList = iexTradingClient.executeRequest(new ChartRequestBuilder().withChartRange(ChartRange.ONE_DAY).withSymbol("AAPL").build());`

### What you expected to happen
Get this data:
https://api.iextrading.com/1.0/stock/aapl/chart/1d

### What actually happened
`com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "marketOpen" (class pl.zankowski.iextrading4j.api.stocks.Chart), not marked as ignorable (24 known properties: "notional", "marketAverage", "high", "open", "marketNumberOfTrades", "marketChangeOverTime", "date", "numberOfTrades", "change", "changeOverTime", "minute", "changePercent", "marketVolume", "marketLow", "volume", "marketNotional", "average", "marketHigh", "unadjustedVolume", "vwap", "low", "close", "label", "simplifyFactor"])`

**Comment by WojciechZankowski** [comment]
Hello,

It was failing because of new fields that has been added to the API by IEX. I fixed it with new version 2.1.2: https://github.com/WojciechZankowski/iextrading4j/releases/tag/IT4J_RELEASE_2_1_2

It will also be available on Maven:

```
<dependency>
	<groupId>pl.zankowski</groupId>
	<artifactId>iextrading4j-all</artifactId>
	<version>2.1.2</version>
</dependency>
```

**Comment by WojciechZankowski** [comment]
Yes, actually I pushed a change with this couple minutes ago. Also I added bunch of tests to keep model up to date with IEX changes.

https://github.com/WojciechZankowski/iextrading4j/commit/90e69e8992a24cd3fa144b6c5d514728446cf9aa#diff-de1dd8f2843a50f803a204e3faee5f50

**Comment by hpdrago** [comment]
Thanks Woj!  You rock!

**Comment by hpdrago** [comment]
Hi Woj, 
Back to my previous question.  My app isn't in production yet, but soon will be.  If IEXTrading adds a new property in their production environment that we don't detect in development, our apps could fail calling IEXTrading.  Do we have any control over this?  Does IEXTrading allow you specify a version?  If not, can't we change the way we receive the JSON data and only grab the information that we want and thus any new fields will not cause our queries to fail?

Thanks,
Mike.

**Comment by WojciechZankowski** [comment]
Hey, 

Yes, we can change the way we receive the JSON. Thats what I did and it is available in version 2.1.3.

Cheers,

Wojtek

**Comment by hpdrago** [comment]
Hi Wojtek,

Excellent!  Thanks!

Mike.
